packages/@lwc/integration-karma/scripts/karma-plugins/lwc.js

```diff
@@ -12,9 +12,8 @@
 'use strict';
 
 const path = require('path');
-
 const { rollup } = require('rollup');
-const lwcRollupPlugin = require('@lwc/rollup-plugin');
```

Comment:
sanity check: are none of the slot-related tests in `@lwc/integration-tests` affected by this slot format change? That project's [build.js](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/integration-tests/scripts/build.js#L10) has its own separate entry point.

Reply:
It's possible that `@lwc/integration-tests` is affected, but I'm hoping the Karma tests are sufficient. The `@lwc/integration-tests` are not terribly well-maintained at this point.